packages/babel-traverse/test/fixtures/rename/parameter-redeclaration/output.js

```diff
@@ -1,9 +1,9 @@
-function f(z, b) {
+function f(a, b) {
```

Comment:
Test case change here.

The test input.js was:

```js
function f(a, b) { // a1
  var a = "redeclared"; // a2
  return b;
}

function g(a) {
  function a() {
    return "function, redeclared";
  }
  return a();
}
```

and a plugin was used on it to rename `a` to `z`.

However, at lines a1 and a2, they refer to different `a`'s, so they definitely shouldnt be both renamed to `z`.